ts_scripts/marsgen.py

```diff
@@ -0,0 +1,121 @@
+import argparse
+import json
+import os
+import sys
+import urllib.request
+
+REPO_ROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..")
+sys.path.append(REPO_ROOT)
+model_store_dir = os.path.join(REPO_ROOT, "model_store")
+os.makedirs(model_store_dir, exist_ok=True)
+
+def generate_mars(mar_config):
+
+    f = open(mar_config, "r")
+    models = json.loads(f.read())
+
+    for model in models:
+        serialized_file_path = None
+        if model.get("serialized_file_remote") and model["serialized_file_remote"]:
+            os.chdir(model_store_dir)
+            serialized_model_file_url = "https://download.pytorch.org/models/{}".format(model["serialized_file_remote"])
+            # Download & create DenseNet Model Archive
+            urllib.request.urlretrieve(serialized_model_file_url, model["serialized_file_remote"])
+            serialized_file_path = os.path.join(model_store_dir, model["serialized_file_remote"])
+        elif model.get("serialized_file_local") and model["serialized_file_local"]:
+            serialized_file_path = model["serialized_file_local"]
+
+        handler = None
+        if model.get("handler") and model["handler"]:
+            handler = model["handler"]
+
+        extra_files = None
+        if model.get("extra_files") and model["extra_files"]:
+            extra_files = model["extra_files"]
+
+        runtime = None
+        if model.get("runtime") and model["runtime"]:
+            runtime = model["runtime"]
+
+        archive_format = None
+        if model.get("archive_format") and model["archive_format"]:
+            archive_format = model["archive_format"]
+
+        requirements_file = None
+        if model.get("requirements_file") and model["requirements_file"]:
+            requirements_file = model["requirements_file"]
+
+        os.chdir(REPO_ROOT)
+        export_path = model_store_dir
+        if model.get("export_path") and model["export_path"]:
+            export_path = model["export_path"]
+
+
+
+        cmd = model_archiver_command_builder(model["model_name"], model["version"], model["model_file"],
+                                             serialized_file_path, handler, extra_files,
+                                             runtime, archive_format, requirements_file, export_path)
+        print(f"## In directory: {os.getcwd()} | Executing command: {cmd}")
+        sys_exit_code = os.system(cmd)
+        if model.get("serialized_file_remote") and model["serialized_file_remote"]:
```

Comment:
I believe this is trying to remove the serialized file as part of a cleanup. Perhaps it'll help to create a separate clean_up function that cleans up any other files as well.

Reply:
a serialized file is the only file which needs to be deleted according to archiver cmd.  A clean up func can be added if more files need to be deleted in the future.